packages/blade/src/components/Checkbox/CheckboxIcon/CheckboxIconWrapperStyles.ts

```diff
@@ -38,10 +38,12 @@ const variants = {
 
 export type CheckboxRectProps = Omit<CheckboxIconProps, 'state'> & {
```

Comment:
We should extract this component out into a common place and not import from Checkbox for Radio

Reply:
Done in PR2